jest-opposite-of-matcher/isNotEqual.js

```diff
@@ -0,0 +1,5 @@
+const isNotEqul = () => {
```

Comment:
`Equl` is spelled wrong. 🙂

Reply:
Yes, thanks. Definitely a silly error. I have corrected it and pushed the change up to this branch. I should have checked that.